assets/svg/gitea-lock-cog.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" aria-hidden="true" viewBox="0 0 14 16">
```

Comment:
```suggestion
<svg width="16" height="16" viewBox="0 0 14 16">
```

I assume this is unneccessary?

Reply:
Actually, I see we do no further postprocessing so it might be required here for now.

Reply:
Checked the functions, `modules/templates/helper.go` and `web_src/js/utils.js` already add the `aria-hidden` attribute to `<svg>` and I think it makes no sense to have it on `<symbol>`, e.g. right now:

````html
<svg class="svg gitea-lock-cog" width="16" height="16" aria-hidden="true">
    <use xlink:href="#gitea-lock-cog">
      <symbol aria-hidden="true" viewBox="0 0 14 16" id="gitea-lock-cog"><path></path></symbol>
    </use>
</svg>